index.html

```diff
@@ -10207,10 +10207,6 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 						<th class="value-name" scope="row">true</th>
 						<td class="value-description">The element is checked.</td>
 					</tr>
-					<tr>
-						<th class="value-name" scope="row"><strong class="default">undefined (default)</strong></th>
```

Comment:
I think we need to allow for undefined to represent that the element is not checkable. This is important for option and treeitem, which may or may not be checkable.

The default value should be defined for the specific roles, not in the property definition. If we want a default for checkbox and radio, it can be defined for those roles. We do not need to say the default value here.

Reply:
aria-checked is required so it cannot have a default value as it is a validation error if it is not specified.

agenda+ for Tomorrow

Reply:
if the element is not checkable, then should it just not have the `aria-checked` attribute?

treeitem and option do not require this attribute.  For treeitem aria-selected is required on the role, and i still am unconvinced that aria-checked should even be allowed on option.